### ISSUE TITLE
Provide an example about how to provide a custom order 

### ISSUE THREAD

**Issue by Unknow-max** [Siilwyn/css-declaration-sorter#87]
Hi!
I have a small question regarding the function that can be passed to `order` option. 
Now, as mentioned in the documentation:

Provide the name of one of the built-in sort orders or a comparison function that is passed to ([`Array.sort`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort)). This function receives two declaration names and is expected to return `-1`, `0` or `1` depending on the wanted order.

 I think that Is pretty clear, however I have hard time figure out how to to do this mainly because I don't know Js, and maybe some peoples can be in my situation too.

It wouldn't be useful to include an example in the readme.md on how to do this?

**Thank you very much!** 🙏

**Comment by Unknow-max** [comment]
@Siilwyn

**Comment by Siilwyn** [comment]
Hey yes, thank you for the reminder @Unknow-max!

I'm planning on creating multiple useful examples and put those in a directory separate from the readme, so there is room for longer examples.

What do you want to achieve? Might be a perfect example to put in.

**Comment by Unknow-max** [comment]
👋,
I want to provide a custom order for sort my CSS properties, as mentioned in the documentation is possibile with the order: option, but I had hard time figure out how to do this (I'm very begginer in js), for that It will be very useful to provide an example of how to to this (maybe I am not the only who don't well js, and also It will be probably a small time saver for all).

**Comment by Siilwyn** [comment]
So ordering with a custom order, I assume you have a list of declarations so an array would be a good fit:
```js
const postcss = require('postcss');
const cssDeclarationSorter = require('css-declaration-sorter');

const myOrder = ['margin', 'border'];

const myCompareFunction = (a, b) => myOrder.indexOf(a) - myOrder.indexOf(b);

postcss([cssDeclarationSorter({order: myCompareFunction})])
  .process('a { border: 0; margin: 0; }')
  .then(result =>
    console.log(result.css === 'a { margin: 0; border: 0; }')
  );
```

Does this work for you as expected and is the example understandable?

**Comment by Unknow-max** [comment]
Yes, working, and it is pretty clear!
I think that this can bee a good example to include in the documentation, maybe adding another property  to the array:

```javascript
const postcss = require('postcss');
const cssDeclarationSorter = require('css-declaration-sorter');

const myOrder = ['margin', 'border', 'padding'];

const myCompareFunction = (a, b) => myOrder.indexOf(a) - myOrder.indexOf(b);

postcss([cssDeclarationSorter({order: myCompareFunction})])
  .process('a { padding: 0; border: 0; margin: 0; }')
  .then(result =>
    console.log(result.css === 'a { margin: 0; border: 0; padding: 0 }')
  );
```

**Comment by Siilwyn** [comment]
Alright cheers! I'll keep this issue open until I add it to documentation examples.

**Comment by Siilwyn** [comment]
Solved with d0aeb565ed032ce1b291758ce422b9ad96669ab7.
